apps/condo/pages/ticket/[id]/index.tsx

```diff
@@ -205,6 +212,22 @@ const TicketIdPage = () => {
     }, {
         fetchPolicy: 'network-only',
     })
+
+    const { objs: comments, refetch: refetchComments } = TicketComment.useObjects({
+        where: { ticket: { id } },
+        // @ts-ignore
+        sortBy: ['createdAt_ASC'],
+    })
+    const updateComment = TicketComment.useUpdate({}, () => {})
+    const deleteComment = TicketComment.useSoftDelete({}, () => {})
+
+    const createCommentAction = TicketComment.useCreate({
+        ticket: id,
+        user: auth.user.id,
+    }, () => { refetchComments() })
+
+    const { link: { role } } = useOrganization()
```

Comment:
role is not required for all users!!

Reply:
Changed in `632e5485`, `604f6fe9`, `6ee0420c`.